docs/en-US/component/carousel.md

```diff
@@ -79,18 +79,18 @@ carousel/vertical
 
 | Name                 | Description                                           | Type    | Accepted Values     | Default    |
```

Comment:
~~Accepted Values~~

The new version of the document has removed the Accepted Values property

Reply:
I look at the latest code, this is not deleted
![image](https://github.com/user-attachments/assets/05124583-a3f0-4ad3-93f7-7a5d174c9a7f)